src/Plugin/EmbedType/Entity.php

```diff
@@ -232,6 +232,10 @@ protected function getEntityTypeOptions() {
         if (!$this->entityTypeManager->getDefinition($entity_type_id)->hasViewBuilderClass()) {
           unset($options[$group][$entity_type_id]);
         }
+        // Filter out entity types that do not support UUIDs.
```

Comment:
I was gonna say:

>This is wrong because there always is one, see `\Drupal\Core\Entity\EntityInterface::uuid`!

But that's apparently not true:

```
   * @return string|null
   *   The UUID of the entity, or NULL if the entity does not have one.
   */
  public function uuid();
```

:(

So +1 for this.